### PR TITLE
Test case and solution for issue #3920

### PR DESCRIPTION
**Pull request by TracyGJG**
Addressed issue #3920 and added a new test case to confirm.
Good find @jgr3go and thanks for giving me an opportunity to practice TDD.
@icambron, your hypothesis as to the probable cause appears to be spot-on, thanks.

### CODE REVIEW

File: src/lib/create/from-string.js [hunk 1]
```diff
@@ -150,8 +150,11 @@ export function configFromRFC2822(config) {
             case 4: // Zone
                 timezone = timezones[match[5]];
                 break;
-            default: // UT or +/-9999
+            case 3: // UT
```
Comment (ichernev) [inline src/lib/create/from-string.js]: Can you please order the case statements by length. Also you can add in comments some examples of each case on the first line.

**Comment by ichernev** [comment]
Thank you for addressing the comment and adding the `toRFC2822String`. Normally these two need to go in separate PRs (one is a feature, the other one a bug fix), changes are in separate file sets so should be easy to split.

**Comment by TracyGJG** [comment]
Not wanting to duplicate effort; were you intending to split the update, apply it as all-in-one, or did you wanting be to raise a feature issue and submit a second PR?

**Comment by ichernev** [comment]
Just add another PR and remove the code from this one. In the changelog different items should come from different PRs (and respectively merges).

**Comment by TracyGJG** [comment]
Separating changes.

**Pull request closed without merging**